plugins/nginx-vhosts/install

```diff
@@ -4,10 +4,12 @@ set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
 case "$DOKKU_DISTRO" in
   ubuntu)
     echo "%dokku ALL=(ALL) NOPASSWD:/etc/init.d/nginx reload, /usr/sbin/nginx -t" > /etc/sudoers.d/dokku-nginx
+    echo "%dokku ALL=(ALL) NOPASSWD:/usr/bin/tail" >> /etc/sudoers.d/dokku-nginx
     ;;
 
   opensuse)
     echo "%dokku ALL=(ALL) NOPASSWD:/sbin/service nginx reload, /usr/sbin/nginx -t" > /etc/sudoers.d/dokku-nginx
+    echo "%dokku ALL=(ALL) NOPASSWD:/usr/bin/tail" >> /etc/sudoers.d/dokku-nginx
```

Comment:
After doing the `plugin` plugin, I remembered that we assume `dokku` will be run by root and thus we `sudo su - dokku` here:  https://github.com/progrium/dokku/blob/63d89942438e2806c5322c2ecddbe2d008e03f42/dokku#L44-L47

Therefore I think we can just filter these commands as well to make them not sudo. I'll create a new PR based on this work.